docs/resources/Channel.md

```diff
@@ -629,22 +634,26 @@ Update a channel's settings. Requires the `MANAGE_CHANNELS` permission for the g
-| permission_overwrites | ?array of [overwrite](#DOCS_RESOURCES_CHANNEL/overwrite-object) objects | channel or category-specific permissions                                                                                                                                        | All                       |
-| parent_id             | ?snowflake                                                              | id of the new parent category for a channel                                                                                                                                     | Text, News, Store, Voice  |
+| Field                 | Type                                                                    | Description                                                                                                                                                                     | Channel Type                                 |
+| --------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| name                  | ?string                                                                 | 2-100 character channel name                                                                                                                                                    | Group DM, Text, News, Store, Voice, Category |
+| type                  | integer                                                                 | the [type of channel](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types); only conversion between text and news is supported and only in guilds with the "NEWS" feature      | Text, News                                   |
+| position              | ?integer                                                                | the position of the channel in the left-hand listing                                                                                                                            | Text, News, Store, Voice, Category           |
+| topic                 | ?string                                                                 | 0-1024 character channel topic                                                                                                                                                  | Text, News                                   |
+| nsfw                  | ?boolean                                                                | whether the channel is nsfw                                                                                                                                                     | Text, News, Store                            |
```

Comment:
don't voice channels have a NSFW attribute on them, even if the client doesn't show this? Might be mistaken here.

Reply:
Yes they do according to the [Guild Voice Channel example](https://discord.com/developers/docs/resources/channel#channel-object-example-guild-voice-channel)

Reply:
perhaps it is time for staff to determine if this is a feature or a missed bug, and we can then document it properly/perhaps request the client also shows this on their trello, or consider it an undocumented artifact that can be removed everywhere?